eclass/coreos-kernel.eclass

```diff
@@ -7,6 +7,16 @@
 # Revision of the source ebuild, e.g. -r1. default is ""
 : ${COREOS_SOURCE_REVISION:=}
 
+# @ECLASS-VARIABLE: COREOS_TARGET_BOARD
+# @DESCRIPTION:
+# Target board name, e.g. "hikey". default is ""
+: ${COREOS_TARGET_BOARD:=}
+
+# @ECLASS-VARIABLE: COREOS_TARGET_DTB
```

Comment:
Is there a particular need for the DTB to be provided by the kernel ebuild? could it just be a independent package?